seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/datatypes/StarRocksTypeConverter.java

```diff
@@ -0,0 +1,648 @@
+/*
```

Comment:
Please add UT for this.

Reply:
ok,thanks!

Reply:
StarRocksTypeConverter referenced from https://github.com/apache/seatunnel/pull/8087